monitoring/quickstart.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// [START monitoring_quickstart]
+// Imports the Google Cloud client library
```

Comment:
I think the biggest use case for the API in code (as opposed to ops-y stuff done in gcloud or stuff done in fluentd) is writing custom metric so we might want to consider making that the quickstart.